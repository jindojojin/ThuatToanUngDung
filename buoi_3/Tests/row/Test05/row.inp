500
13 13 5
10 5 3
19 17 4
10 10 4
27 2 4
7 25 1
13 10 3
7 6 5
26 22 5
4 4 4
17 19 5
27 20 2
13 17 5
4 2 4
9 12 3
9 13 2
4 26 1
24 10 1
16 9 5
3 28 2
2 10 4
26 9 4
25 16 2
25 3 5
2 25 1
14 6 2
25 21 5
13 3 1
19 6 4
5 1 3
24 10 2
28 9 3
24 28 3
24 25 5
19 29 5
16 28 5
14 2 5
12 13 1
13 25 1
7 12 5
28 19 2
16 23 1
9 13 1
9 11 1
15 13 5
15 20 1
16 2 5
27 22 1
5 12 4
8 12 3
14 11 3
13 8 3
13 20 1
7 14 1
18 22 2
24 12 1
23 19 1
21 9 5
14 13 1
10 14 1
8 20 3
21 10 3
20 14 2
27 15 5
10 26 5
26 5 3
22 13 1
26 12 1
25 29 4
13 22 2
17 16 5
20 22 4
11 20 5
19 25 1
6 8 3
23 7 1
5 19 5
25 15 2
12 3 1
9 9 2
17 26 4
29 13 3
24 16 4
27 9 5
4 29 5
15 18 2
22 2 3
7 2 4
6 17 3
18 1 2
15 19 3
10 12 3
12 29 4
16 6 4
24 28 5
14 24 5
12 8 3
8 11 3
12 21 1
2 28 4
25 17 3
13 19 3
23 10 5
29 28 3
8 10 2
29 23 1
5 28 2
11 28 4
20 13 3
10 8 1
9 5 2
25 17 2
25 11 1
11 24 3
18 15 1
12 10 3
12 18 5
1 3 1
7 16 3
9 7 4
16 14 5
9 4 1
19 1 5
18 11 3
23 4 5
24 7 3
22 24 2
22 10 3
17 5 4
20 29 4
10 1 4
3 24 2
1 1 1
28 1 2
9 18 1
1 13 4
27 20 3
17 20 3
15 27 1
18 24 4
28 14 3
2 3 2
25 16 1
18 6 4
9 25 1
16 8 4
8 8 3
8 10 4
6 4 4
5 18 4
18 28 3
14 23 5
22 29 4
7 19 4
20 23 3
28 14 2
25 1 3
20 3 2
6 18 2
15 29 3
29 19 3
4 26 3
12 6 5
18 26 2
22 7 4
8 11 5
26 10 4
2 21 1
3 12 4
3 18 1
23 7 1
14 28 2
27 6 3
6 13 4
24 5 5
6 14 3
2 15 5
1 23 2
8 10 1
11 6 5
9 4 3
15 7 5
19 14 5
3 18 3
21 8 4
24 29 4
29 17 4
18 28 1
21 25 4
10 28 1
7 15 5
6 16 1
19 25 1
15 10 4
2 20 1
26 24 1
6 9 4
1 7 3
7 1 3
10 27 1
16 13 5
21 5 3
25 8 5
20 10 1
4 7 1
2 1 4
19 13 3
19 13 1
1 15 3
11 25 5
22 1 5
11 17 3
8 7 1
15 7 4
8 29 1
27 8 2
1 2 3
16 24 5
29 23 5
19 15 5
17 20 3
3 6 5
9 6 5
21 12 1
12 2 1
22 22 4
26 11 2
22 9 3
12 14 5
21 19 2
21 22 5
5 28 5
27 26 2
9 3 2
26 8 3
8 20 2
3 10 1
29 27 3
22 10 3
22 5 1
26 2 4
29 29 2
29 1 3
9 27 1
15 5 1
21 7 4
3 23 4
7 1 5
4 14 4
27 3 1
16 10 4
13 28 5
4 25 5
10 14 2
1 29 2
15 28 4
28 15 5
29 29 5
20 24 4
16 21 2
4 27 5
1 26 1
23 7 3
23 18 1
13 21 3
27 18 5
12 25 2
5 19 3
2 5 5
10 14 5
2 10 3
20 8 5
8 17 1
12 6 2
12 18 3
24 15 1
8 20 5
26 8 5
16 25 1
27 6 4
9 18 5
16 2 4
26 16 4
2 20 5
3 14 5
15 26 3
14 21 1
19 14 3
20 15 2
8 4 5
25 4 2
6 6 4
1 1 5
19 18 4
9 20 4
26 19 1
10 2 5
25 3 5
10 5 2
16 16 1
2 20 4
16 12 3
18 18 5
7 14 5
15 11 1
13 23 4
1 19 4
1 14 4
15 14 4
20 14 2
23 21 2
17 23 2
17 16 1
3 13 1
16 18 1
1 1 1
1 2 4
3 21 5
2 19 1
21 22 1
25 16 1
24 18 2
26 26 1
4 23 1
16 29 2
7 19 4
14 24 1
21 10 2
17 1 4
4 14 4
20 2 5
10 20 4
2 17 3
7 20 3
4 26 4
28 28 1
24 26 2
21 9 4
3 16 3
24 23 4
6 18 1
26 14 2
3 11 4
3 16 4
24 17 4
10 3 1
3 27 1
19 2 3
5 26 3
17 18 4
7 26 1
25 20 1
18 3 4
27 10 3
20 10 4
12 9 3
26 11 3
19 8 2
21 29 5
29 28 2
28 29 3
4 19 1
28 28 2
27 1 1
21 23 2
26 17 3
5 10 4
19 29 3
4 20 3
3 22 5
29 13 4
13 29 5
26 28 4
11 6 5
3 14 4
4 7 1
22 26 1
1 9 5
25 20 5
13 23 4
18 14 1
19 16 4
4 25 3
25 21 1
25 18 2
24 5 3
10 10 5
20 9 4
4 17 3
23 29 1
24 7 3
3 18 4
29 5 4
3 3 4
27 21 1
12 12 1
16 14 5
5 24 3
1 20 5
23 19 4
10 8 1
7 8 4
21 21 2
23 24 4
3 18 1
3 25 1
23 6 2
24 3 2
27 26 1
1 17 3
5 17 3
25 26 4
8 10 1
27 25 1
26 18 3
26 14 3
8 23 3
26 17 2
14 4 4
5 10 1
5 6 4
18 19 2
26 24 3
3 1 5
5 27 1
13 14 5
17 21 4
9 16 5
26 17 2
24 6 3
29 14 3
17 17 3
28 7 5
6 12 3
9 17 5
11 16 4
1 14 3
27 25 2
8 11 3
23 19 5
17 26 3
6 22 4
20 26 3
19 29 1
26 4 4
7 1 5
4 27 3
27 28 5
9 7 4
16 8 2
25 17 2
13 18 3
20 24 1
3 15 5
10 28 2
9 1 3
21 19 4
19 26 1
11 5 1
21 23 5
26 7 3
15 2 4
26 12 5
17 17 2
13 13 3
26 29 2
9 20 5
5 8 1
25 23 3
27 1 2
5 6 3
12 23 4
24 11 2
18 1 2
17 13 4
6 11 2
2 10 4
10 3 1
23 25 4
9 10 2
12 18 5
12 6 5
14 7 1
27 6 3
5 2 4
9 29 4
27 2 5
27 27 4
19 26 2
5 16 4
29 15 1
9 14 5
11 19 1
8 26 3
21 5 1
29 29 2
3 13 3
10 6 5
12 7 3
20 1 3
100
283 500 4
123 452 1
41 90 2
58 380 4
167 355 4
290 349 4
39 65 5
74 216 5
329 485 1
229 230 3
8 261 1
30 68 3
333 426 1
122 332 2
394 497 4
18 376 2
286 453 1
177 331 3
73 348 5
180 340 1
188 464 2
136 187 2
306 475 4
201 418 3
84 328 5
255 477 3
158 361 5
287 319 5
251 499 4
223 366 1
102 448 1
169 241 5
243 248 4
359 431 5
229 245 2
16 232 2
367 467 5
147 393 1
101 102 3
111 301 3
78 396 5
317 439 2
137 438 3
231 416 1
43 90 3
27 71 5
62 416 1
121 207 2
149 460 3
178 314 2
69 229 4
187 392 2
390 477 4
147 413 1
51 430 2
177 186 3
140 362 4
199 491 3
402 458 3
448 465 2
3 303 2
95 99 3
8 302 4
150 404 2
142 203 3
146 211 3
54 283 3
99 115 1
248 294 4
164 256 3
278 472 2
233 342 3
61 327 4
142 436 3
23 482 5
145 408 4
29 364 2
50 249 2
9 249 3
350 391 1
130 208 1
264 303 2
266 420 5
289 294 2
163 454 5
104 244 2
239 428 3
184 332 4
2 135 1
113 464 4
62 291 1
316 413 4
9 481 3
238 460 2
52 235 3
26 210 3
94 240 3
107 448 4
318 325 5
195 291 2
