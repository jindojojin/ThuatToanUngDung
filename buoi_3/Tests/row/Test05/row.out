10844
17491
1706
15334
8227
2882
1165
4122
8142
0
12047
1778
6394
8295
5214
15250
11238
4051
10201
9800
13561
1449
8272
8580
9823
10152
8245
613
12354
7752
18822
3575
216
1691
2129
6714
4260
13185
247
8602
11833
7327
13354
11897
1425
3064
19608
2077
13959
5112
7677
9887
3767
14887
16237
90
9842
12648
4068
1122
10428
184
11446
11771
2598
2860
11350
1292
2104
2684
10834
3075
11671
12608
19242
12967
14199
7104
11976
2829
4294
822
3546
100
11458
5398
6899
6219
6988
16255
9880
6185
22128
11482
9553
9901
9080
16071
0
3908
